ui/views.py

```diff
@@ -168,7 +169,11 @@ def welcome(request):
     return render(
         request,
         "welcome.html",
-        {"repos": get_repos(request.user.id)}
+        {
+            "repos": get_repos(request.user.id),
+            "support_email": get_var('EMAIL_SUPPORT',
+                                     'support@example.com')
```

Comment:
I think you need to take the value from the setting and not from the environment
So if you import the settings
```
...
from django.conf import settings
...
```
then your code is just:
```
...
"support_email": settings.EMAIL_SUPPORT,
...
```